### PR TITLE
fix failed to rotate log for container for docker runtime

### PR DESCRIPTION
**Pull request by wackxu**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

when the container log is big enough to rotate and then edgecore is always print the err as below

```
May 30 01:54:15 starstarblaze edgecore[20058]: E0530 01:54:15.618504   20058 container_log_manager.go:259] "Failed to rotate log for container" err="failed to rotate log \"/var/log/pods/default_test-7kfx6_b78dea8a-f7f2-4cdb-afee-0a1933958259/test/1.log\": failed to reopen container log \"0da4e3b9dc87f77ded3c1aba9b861f2a89f224fea856efe79deb4058e0ac5376\": rpc error: code = Unknown desc = docker does not support reopening container log files" path="/var/log/pods/default_test-7kfx6_b78dea8a-f7f2-4cdb-afee-0a1933958259/test/1.log" containerID="0da4e3b9dc87f77ded3c1aba9b861f2a89f224fea856efe79deb4058e0ac5376"
```

due to docker does not support reopening container log files and  Container log manager only starts when the container runtime is remote (not docker), because we can't implement ReopenContainerLog for docker.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/pkg/edged/edged.go [hunk 1]
```diff
@@ -543,14 +545,17 @@ func newEdged(enable bool) (*edged, error) {
 		return nil, err
 	}
 
-	// create a log manager
-	logManager, err := logs.NewContainerLogManager(runtimeService, ed.os, "10Mi", 5)
-	if err != nil {
-		return nil, fmt.Errorf("new container log manager failed, err: %s", err.Error())
+	if ed.containerRuntimeName == remoteContainerRuntime {
```
Comment (gy95) [inline edge/pkg/edged/edged.go:548]: can be replaced by constant `RemoteContainerRuntime` in package "k8s.io/kubernetes/pkg/kubelet/types"

**Review by gy95 — APPROVED** [review]
/lgtm

File: edge/pkg/edged/edged.go (hunk 1 above)
Comment (gy95) [reply edge/pkg/edged/edged.go]: /lgtm

**Comment by gy95** [comment]
/lgtm 
need merged :)

**Review by vincentgoat — COMMENTED** [review]
lgtm

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**